### PR TITLE
MB-14982 Rename all the usages of master into main where it makes sense

### PR DESCRIPTION
**Pull request by YanZ777**
[MB-14982](https://dp3.atlassian.net/browse/MB-14982)

This PR mostly fixes links. Some of the links have been changed to point to a doc already in this repo.
Change sentences to use the word main instead.
Adjust ADR references so that it attaches a name and number instead of just saying "ADR".
The links that I haven't changed are for things that specifically link to stuff that's not ours, e.g. golang documentation or Storybook.

There is one link that still needs to be updated once #268 is merged.
```
*If the `pre-commit` command is not found or errors out, please make sure you have the [pre-requisites](https://github.com/transcom/mymove/blob/master/README.md#setup-prerequisites) installed.*
```

I am opting not to change prior ADR language since I have not seen retroactive edits made to them before. 
[ADR 0047](https://transcom.github.io/mymove-docs/docs/adrs/build-only-pull-requests-in-circleci)

I am opting not to change the verbage inside of the Open Telemetry file since I believe it's a direct quote.
https://transcom.github.io/mymove-docs/docs/backend/guides/open-telemetry#elasticache-redis
```
- IsMaster
- KeyBasedCmds
- KeyBasedCmdsLatency
- MasterLinkHealthStatus
```

## Things I'd like reviewers to specifically look out for
Do the links look good?
Did I change anything I shouldn't have?

There is a follow on ticket to address mystery links. [MB-15177](https://dp3.atlassian.net/browse/MB-15177)

~~- pkg/handlers/contexts.go https://github.com/transcom/mymove/blob/master/pkg/handlers/contexts.go~~

~~https://transcom.github.io/mymove-docs/docs/backend/guides/access-global-variables#setting-up-global-variables-in-the-handler-config~~

[MB-14982]: https://dp3.atlassian.net/browse/MB-14982?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MB-15177]: https://dp3.atlassian.net/browse/MB-15177?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by kctruss — APPROVED** [review]
So good! So clean! 😻 

EDIT: I didn't catch that this was still in draft 😬 but I still approve!!

File: docs/backend/guides/access-global-variables.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ We use command line flags to get the environment variables. The flags are set in
 
 To add an application variable to the handler config, we create essentially a getter and setter in the handler config.
 (Ex. `SetUseSecureCookie` and `UseSecureCookie`)
-Follow the pattern in [pkg/handlers/contexts.go](https://github.com/transcom/mymove/blob/master/pkg/handlers/contexts.go)
+Follow the pattern in [pkg/handlers/config.go](https://github.com/transcom/mymove/blob/main/pkg/handlers/config.go)
```
Comment (YanZ777) [inline docs/backend/guides/access-global-variables.md:32]: Could someone verify that this the correct file to link to? I think this what `contexts.go` got moved/renamed to.

Comment (rogeruiz) [reply docs/backend/guides/access-global-variables.md:32]: Yes exactly correct. It was renamed in this commit ae1c895d871099f6fc4fe3ee8805892602f0e169. You check out the bottom of this page for proof.

[=> this page shows the history of the file `config.go`](https://github.com/transcom/mymove/commits/main/pkg/handlers/config.go)

**Comment by YanZ777** [comment]
I'm going to create a separate ticket to address all the mystery links.
This PR is going to get taken out of draft and merged.

**Pull request merged**